padertorch/train/hooks.py

```diff
@@ -39,18 +38,18 @@ class Priority(IntEnum):
     Summary 50
     Print 40 NotImplemented
     ProgressBar(TQDM) 30 NotImplemented
-    Validation 25
-    Checkpoint 20
+    Validation 20
+    Checkpoint 11
     End 10
 
     End has to be the last one
     Summary before Validation, clears timer information
     Print and ProgressBar may access Summary
     """
     END = 10
+    CHECKPOINT = 11
```

Comment:
Should we write somewhere a motivation, why CHECKPOINT is the pre last Hook?

Reply:
Yes. Done.